pulp_container/app/tasks/sync_stages.py

```diff
@@ -131,6 +131,7 @@ async def run(self):
                     )
                     await self.put(man_dc)
                     tag_dc.extra_data["tagged_manifest_dc"] = man_dc
+                    await man_dc.resolution()
```

Comment:
```suggestion
                    await man_dc.resolution()
                    assert man_dc.content.pk is not Null
```
Can you add debug statements like this?